docs/06-manual-builds.md

```diff
@@ -0,0 +1,136 @@
+# Manual builds
+
+If you want to create `.ipa` or `.apk` builds for testing purposes you can follow this guide. If you only want to quickly push an update to the stores then take a look [at the CI/CD documentation](08-cicd.md#triggering-a-build-in-ci).
+
+## Manually building for iOS
```

Comment:
Imo this section is too complicated if I just want to run it locally on my device/on the emulator (which should be the main usecase here). At least the section to run it in the emulator/local device should be first imo.

Reply:
I think the corresponding commands should also be part of getting started of the readme. If I am a new contributor to the project, I don't want to know all the details of fastlane, certificates.... but rather just run the app and start developing.

Reply:
I tried to restructure it a bit. But it still is a little bit complicated.

I already linked it from the README.